frontend/src/modules/integration/components/integration-list.vue

```diff
@@ -304,13 +332,17 @@ export default {
               'github',
               'slack',
               'discord',
-              'twitter'
+              'twitter',
+              'devto'
             ].includes(i.platform)
           )
         : integrationsJsonArray,
       twitter: {
         popover: false,
         hashtags: []
+      },
+      devto: {
```

Comment:
Maybe we could manage this visibility state inside the `devto-integration-widget.vue`, since we have a proper component for it, and leave this file with one less thing to manage.

Reply:
Agree - will change.